core-bundle/src/Doctrine/Schema/DcaSchemaProvider.php

```diff
@@ -64,7 +64,7 @@ public function appendToSchema(Schema $schema): void
 
             if (isset($definitions['SCHEMA_FIELDS'])) {
                 foreach ($definitions['SCHEMA_FIELDS'] as $fieldName => $conf) {
-                    if ($table->hasColumn($fieldName)) {
+                    if ($table->hasColumn($conf['name'] ?? $fieldName)) {
```

Comment:
This was a necessary change due to the `DcaSchemaProviderTest`. @ausi can you may be take a look why sometimes the `$definitions['SCHEMA_FIELDS']` array has the name of the field as the key and sometimes only an index with the name stored within the array value?